digestset/deprecated.go

```diff
@@ -0,0 +1,51 @@
+package digestset
+
+import (
+	"github.com/opencontainers/go-digest"
+	"github.com/opencontainers/go-digest/digestset"
+)
+
+// ErrDigestNotFound is used when a matching digest
+// could not be found in a set.
+//
+// Deprecated: use [github.com/opencontainers/go-digest/digestset.ErrDigestNotFound].
```

Comment:
FYI: [the fully-qualified package path is not required for doc links to imported packages.](https://go.dev/doc/comment#links:~:text=When%20referring%20to,cannot%20be%20used.)
```suggestion
// Deprecated: use [digestset.ErrDigestNotFound].
```

Reply:
> FYI: [the fully-qualified package path is not required for doc links to imported packages.](https://go.dev/doc/comment#links:~:text=When%20referring%20to,cannot%20be%20used.)

Ah, yes, I actually knew that. I had that initially, but had to wiggle the code a bit to make the deprecations apply. Reason was that a _group of vars_ won't pick up the deprecation, so; something like below doesn't work;

```go
var (
    // Foo bla bla
    //
    // Deprecated: blablabla
    Foo string

    // Bar bla bla
    //
    // Deprecated: blablabla
    Bar string
)
```

I tried if changing some of the Deprecation line would work, then forgot to change it back 🙈 
 

Planning to open a bugreport for that in the Go issue tracker.